src/app/domain/backup/services/uploadMedia.ios.ts

```diff
@@ -2,44 +2,38 @@
 import { CameraRoll } from '@react-native-camera-roll/camera-roll'
 import RNFileSystem from 'react-native-fs'
 
-import { Media } from '/app/domain/backup/models/Media'
+import { Media, UploadMediaResult } from '/app/domain/backup/models/Media'
 
-import type CozyClient from 'cozy-client'
+import CozyClient, { StackErrors, IOCozyFile } from 'cozy-client'
 
-interface UploadMediaResult {
-  success: boolean
-  data?: object
+const getRealFilepath = async (media: Media): Promise<string> => {
+  const data = await CameraRoll.iosGetImageDataById(media.path)
+
+  let filepath = data.node.image.filepath
+
+  if (filepath === null) {
+    throw new Error('Impossible to get a real filepath')
+  }
+
+  filepath = filepath.replace('file://', '')
+
+  if (media.type === 'image') {
+    return filepath
+  } else if (media.type === 'video' && media.subType === 'PhotoLive') {
+    // We determine real filepath for the video part of a Live Photo
```

Comment:
nit: this comment is not present in the `getMedias` code which does the same thing. Maybe we can extract this in a `getVideoPathFromLivePhoto` method so we don't need to explain it everywhere?